project/Dependencies.scala

```diff
@@ -17,7 +17,7 @@ object Dependencies {
   lazy val slf4jVersion = "1.7.36"
   lazy val awsSdkVersion = "1.11.1033"
   val tensorflowJavaVersion = "0.4.0" // Match Tensorflow 2.7.0 https://github.com/tensorflow/java/#tensorflow-version-support
-  val xgboostVersion = "1.6.1"
+  val xgboostVersion = "1.7.3"
```

Comment:
Let's see what happens with the latest version

Reply:
nice!